src/main/kotlin/com/team4099/robot2021/config/Constants.kt

```diff
@@ -31,4 +33,28 @@ object Constants {
     const val THROTTLE_DEADBAND = 0.04
     const val TURN_DEADBAND = 0.035
   }
+
+  object Intake {
+    const val INTAKE_MOTOR = 0 //temp
+    const val ARM_SOLENOID_PORT_1 = 0 //temp
+    const val ARM_SOLENOID_PORT_2 = 1 //temp
+    const val TAB = "Intake"
+    const val INTAKE_STATE = "IntakeState"

```

Comment:
no reason to have this in constants i think

Reply:
shouldn't the arm solenoid ports still be constants?